packages/layout/src/composable/useLayout.ts

```diff
@@ -129,76 +206,73 @@ const getFixedPanelsStatus = () => {
   return { leftPanelFixed, rightPanelFixed }
 }
 
-const closeSetting = (forceClose) => {
+const closeSetting = (forceClose?: boolean) => {
   const { settings } = layoutState
   if (!settings.fixedPanels.includes(settings.render) || forceClose) {
-    settings.render = null
+    settings.render = ''
   }
 }
 
 // 激活plugin面板并返回当前插件注册的Api
-const activePlugin = (name, noActiveRender) => {
+const activePlugin = (name: string, noActiveRender: boolean) => {
```

Comment:
`noActiveRender` 是可选参数
```suggestion
const activePlugin = (name: string, noActiveRender?: boolean) => {
```

Reply:
已修改